sheepdog/transactions/upload/__init__.py

```diff
@@ -323,3 +333,18 @@ def handle_xml_transaction(role, program, project, parser, **tx_kwargs):
 
     return _single_transaction(
         role, program, project, name, 'XML', original, data, **tx_kwargs)
+
+
+def get_active_data_release(g):
+    """
+    Args:
+        g (psqlgraph.PsqlGraphDriver):
+    Returns:
+        str: data release string
+    """
+    with g.session_scope():
+        try:
+            release = g.nodes(models.DataRelease).props(released=False).one()
+            return "{}.{}".format(release.major_version, release.minor_version)
+        except NoResultFound or MultipleResultsFound:
```

Comment:
you probably want to do `except (NoResultFound, MultipleResultsFound)`

Reply:
this also should never happen, because we have checks in `gdcapi` iirc... But if we actually end up here, do you think that raising a `UserError` here is better, than returning `None` and then things go sideways when we try to do `StrictVersion(None)` and get a not-so-informative error message?